cluster-autoscaler/utils/nodegroupset/compare_nodegroups.go

```diff
@@ -124,6 +124,12 @@ func IsNodeInfoSimilar(n1, n2 *schedulercache.NodeInfo) bool {
 			if label == kubeletapis.LabelGardenerWorkerPool {
 				continue
 			}
+			if label == "topology.kubernetes.io/zone" {
+				continue
+			}
+			if label == "topology.kubernetes.io/region" { // TODO: Rebase to latest CA
```

Comment:
Please change these into `StringConstants`. Ideally would be good to have something like this - https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go#L42-L50. However, `StringConstants` should be okay.

Reply:
done. 
Unfortunately, the constants in mcm-branch are coming from the vendored kubelet, and I dint want to update the vendor only to add the topology constants there. The right approach is to re-vendor the k8s version. The link you posted is from the master branch, code-path is different than our fork. 

As I anyways added constants in the current file, also migrating the gardener-specific constants here to nullify the previous change in the vendor.